### ISSUE TITLE
No Install Docs

### ISSUE THREAD

**Issue by conor-f** [bitfieldaudio/OTTO#182]
There's no installation/testing docs to be found? Maybe I'm just looking in the wrong place!

**Comment by topisani** [comment]
You're right, there's not. Currently, there's not a lot to test without the hardware anyway, hence why it's missing
